pulp_2_tests/tests/rpm/cli/test_sync.py

```diff
@@ -220,6 +265,35 @@ def get_rpm_names(cfg, repo_id):
     ]
 
 
+def list_units(cfg, unit_type):
+    """Return a list of units in ``/var/lib/pulp/content/units/``.
+
+    The unit type can be a number of different types.
+
+    Examples are:
+    * distribution
+    * rpm
+    * srpm
+    * drpm
+    * modulemd
+    * modulemd_defaults
+
+    This method should be extensible to take any ``unit_type`` and
+    operate as required.
+    """
+    try:
```

Comment:
@bherrin3, as we talked, let`s evaluate an alternative solution and avoid the use of try/except. Perhaps use a context manager that certain expected exception was raised, and so on.